tasks/59_unite-unique/unite-unique.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef */
+/* eslint-disable prefer-rest-params */
+export function uniteUnique() {
+  let array = [];
+
+  for (let i = 0; i < arguments.length; i++) {
+    array = array.concat(arguments[i]);
+  }
+
+  let uniqueArray = Array.from(new Set(array));
```

Comment:
Let's try to do it without set :) Also, we don't to create `uniqueArray`, we can do direct return  `return Array.from(new Set(array));`